azure-pipelines.yml

```diff
@@ -204,38 +204,34 @@ stages:
         -TsaCodebaseName "Arcade-Validation"
         -TsaPublish $True'
 
-  - stage: Push_to_latest_channel
-    displayName: Push build to 'Latest' channel
-    dependsOn:
-      # This will run only after all the publishing stages have run.
-      # These stages are introduced in the eng/common/templates/post-build/channels YAML templates
-      - NetCore_Dev5_Publish
-      - Net5_Preview1_Publish
-      - Net5_Preview2_Publish
-      - Net_Eng_Latest_Publish
-      - Net_Eng_Validation_Publish
-      - General_Testing_Publish
-      - NETCore_Tooling_Dev_Publishing
-      - NETCore_Tooling_Release_Publishing
-      - NET_Internal_Tooling_Publishing
-      - NETCore_Experimental_Publishing
-      - Net_Eng_Services_Int_Publish
-      - Net_Eng_Services_Prod_Publish
-    jobs:
+# Arcade validiation with additional repos
+- stage: Additional_Arcade_Validation
+  displayName: Additional Arcade Validation
+  jobs:
     - template: /eng/common/templates/job/job.yml
       parameters:
-        name: Push_to_latest_channel
```

Comment:
I think this will save you some pain in the future if you declare a job matrix here so that we can just keep adding additional repos. I won't block on it this time, but next

Reply:
Agreed. I was going to configure that when I did the runtime repo.

Reply:
Add this as a comment in core-eng#9283